src/metrics/site_level.jl

```diff
@@ -23,7 +23,11 @@ function per_loc(metric, data::NamedDimsArray{D,T,N,A})::NamedDimsArray where {D
     #   see: https://stackoverflow.com/a/62040897
     tf = axes(data, :timesteps)
     s::Vector{eltype(data)} = map(metric,
-        JuliennedArrays.Slices(data[timesteps=tf], dim(data, :timesteps), dim(data, :scenarios))
+        JuliennedArrays.Slices(
+            data[timesteps=tf],
+            NamedDims.dim(data, :timesteps),
+            NamedDims.dim(data, :scenarios)
```

Comment:
How about we standardize to `axiskeys()`

```suggestion
            axiskeys(data, :timesteps),
            axiskeys(data, :scenarios)
```